tests/validation/tests/v3_api/test_istio.py

```diff
@@ -326,18 +428,70 @@ def validate_no_access(url, user):
     return response
 
 
-def update_istio_app(answers, user):
+def validate_all_workloads(deployment_list=None, daemonset_list=None,
```

Comment:
Move this to `common.py` - rename to signify that it's checking `rancher` prefix